ui/src/Components/Dashboard/Table/TableHeader/SortCursor.js

```diff
@@ -0,0 +1,34 @@
+import { useState } from "react";
+import { TABLE } from "../../../../Constant/constant";
+
+export default function SortCursor(props) {
+  const [currentClickedCursor, setCurrentClickedCursor] = useState("");
```

Comment:
We should set the useState initial NULL instead of a blank string.

Reply:
@Neha  Any particular reason for this?

Reply:
good question: Just I am biased towards NULL. Can I request you to search on this, please?